x/qgb/keeper/keeper.go

```diff
@@ -13,17 +12,59 @@ import (
 type Keeper struct {
 	cdc      codec.BinaryCodec
 	storeKey sdk.StoreKey
-	memKey   sdk.StoreKey
+
+	// FIXME: should these modules be added now ? Or we can keep them until later along with slashing? Probably, we would need some of them when adding tests
+	//bankKeeper     *bankkeeper.BaseKeeper
+	//StakingKeeper  *stakingkeeper.Keeper
+	//SlashingKeeper *slashingkeeper.Keeper
+	//DistKeeper     *distrkeeper.Keeper
+	//accountKeeper  *authkeeper.AccountKeeper
 }
 
-func NewKeeper(cdc codec.BinaryCodec, storeKey, memKey sdk.StoreKey) *Keeper {
+func NewKeeper(cdc codec.BinaryCodec, storeKey sdk.StoreKey) *Keeper {
 	return &Keeper{
 		cdc:      cdc,
 		storeKey: storeKey,
-		memKey:   memKey,
 	}
 }
 
 func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 	return ctx.Logger().With("module", fmt.Sprintf("x/%s", types.ModuleName))
 }
+
+// prefixRange turns a prefix into a (start, end) range. The start is the given prefix value and
+// the end is calculated by adding 1 bit to the start value. Nil is not allowed as prefix.
+// 		Example: []byte{1, 3, 4} becomes []byte{1, 3, 5}
+// 				 []byte{15, 42, 255, 255} becomes []byte{15, 43, 0, 0}
+//
+// In case of an overflow the end is set to nil.
+//		Example: []byte{255, 255, 255, 255} becomes nil
+// MARK finish-batches: this is where some crazy shit happens
+// FIXME: should we keep these funny comments ?
```

Comment:
we can obv keep the informative parts, but yeah I don't think we need the silly parts